sswlib/src/main/java/states/ifEngine.java

```diff
@@ -41,6 +41,7 @@
     public String BookReference();
     public int GetFullCrits();
     public boolean IsFusion();
+    public boolean IsFission();
     public boolean IsNuclear();
```

Comment:
What is the "IsNuclear" property used for?  I would have assumed that is for fission engines.

Reply:
I'm not sure what that distinction is for, maybe @Maelwys would know...

Fusion and Fission are both nuclear reactions so IsNuclear would also logically apply to Fusion engines, but looking through the code I don't think IsNuclear() is ever called for any of the states.

Reply:
Scratch that, just found this in frmVee:

```
// only nuclear-powered mechs may use jump jets
            if (CurVee.GetEngine().IsNuclear()) {
                /*
                 * if( cmbJumpJetType.getSelectedItem() == null ) {
                 * EnableJumpJets( false ); } else { EnableJumpJets( true ); }
                 */
            } else {
                EnableJumpJets(false);
            }
            return;
```
So I guess jump jets can only be used on mechs with nuclear engines? In any case that should apply to both fusion and fission.